auth/auth_test.go

```diff
@@ -182,6 +202,9 @@ func TestCustomTokenInvalidCredential(t *testing.T) {
 }
 
 func TestVerifyIDToken(t *testing.T) {
+	if appengine.IsDevAppServer() {
```

Comment:
I'm a little curious as to why this is needed. What's causing this test to fail in dev app server? If it's a legit reason, recommend adding a comment here to state that.